src/main/kotlin/com/jobtracker/api/repository/entities/_Constants.kt

```diff
@@ -1,2 +1,2 @@
-const val DEFAULT_FIELD_LENGTH = 256
-const val DEFAULT_NOTES_LENGTH = 2048
\ No newline at end of file
+const val DEFAULT_FIELD_LENGTH = 255
+const val DEFAULT_NOTES_LENGTH = 2047
```

Comment:
The lengths that I put in the database schema were 256 and 8192. Do you mind matching those values?